source/style-guide/markup/directives/metadata.txt

```diff
@@ -1,20 +1,30 @@
+.. _metadata:
+
 ========================
 Page Metadata Directives
 ========================
 
+.. default-domain:: mongodb
+
 meta
 ----
 
-Generates HTML ``<meta>`` tags.
+Use ``.. meta::`` to add |html| meta tags to a page. Adding meta tags 
+aids in searches both internally and via the Internet. You can add both 
+keywords and descriptions. To learn more, see :ref:`seo-guidelines`.
 
 .. code-block:: rst
+	     
+	   .. meta::
+	      :keywords: read concern, local read concern, read isolation, transactions, multi-document transactions
+	      :description: You can tune the consistency and availability of your application using write concerns and read concerns.
 
-   .. meta::
+Add the ``.. meta::`` directive on the line below the 
+``.. default-domain::`` directive with a line break in between.
```

Comment:
Suggestion:

The default-domain directive doesn't do anything anymore (ignored in [ComponentFactory.js](https://github.com/mongodb/snooty/blob/master/src/components/ComponentFactory.js#L88)), so it might be helpful just to mention placing the meta directive directly after the title of the page.

Similarly, you can omit it from the new pages in this PR.